internal/jimm/model.go

```diff
@@ -577,14 +577,8 @@ func (j *JIMM) AddModel(ctx context.Context, user *openfga.User, args *ModelCrea
 		return nil, errors.E(op, err)
 	}
 
-	// fetch user model defaults
-	userConfig, err := j.IdentityModelDefaults(ctx, user.Identity)
-	if err != nil && errors.ErrorCode(err) != errors.CodeNotFound {
-		return nil, errors.E(op, "failed to fetch cloud defaults")
-	}
-	builder = builder.WithConfig(userConfig)
-
 	// fetch cloud defaults
+	// TODO(SimoneDutto): we should get the implicit cloud and then get the defaults.
```

Comment:
yes, if there is only one cloud known to JIMM. otherwise there is no such thing as a default cloud (i think)

Reply:
i am working on this